scripts/verify_jit_cache_abi.py

```
@@ -0,0 +1,177 @@
+    tag = f"cp{sys.version_info.major}{sys.version_info.minor}"
+    wheels = [
+        u for u in urls if u["filename"].endswith(".whl") and "linux" in u["filename"]
+    ]
+    # abi3 wheels serve every interpreter; otherwise match this one.
+    match = [
+        u for u in wheels if "abi3" in u["filename"] or tag in u["filename"]
+    ] or wheels
+    if not match:
+        sys.exit(f"apache-tvm-ffi {version} has no linux wheel to inspect")
+    dest = workdir / match[0]["filename"]
+    urllib.request.urlretrieve(match[0]["url"], dest)
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)